app/code/Magento/ConfigurableProduct/view/adminhtml/web/js/components/associted_product_list.js

```diff
@@ -0,0 +1,11 @@
+define([
+    'uiComponent',
+    'ko'
+], function (Component, ko) {
+    "use strict";
+    return Component.extend({
+        defaults: {
+            isShowAddProductButton: ko.observable(false)
```

Comment:
Use `initObservable` method to initialize observable.

Reply:
@dmytro-ch  i tried but some how not working so i am keeping same